docs/demo.rst

```diff
@@ -0,0 +1,49 @@
+Demo
+====
+
+The spirit of this demo is for you to get a feel for IRC Hooky without
+investing any time in setting it up and deploying it yourself.
+
+It basically works as follows:
+
+- Add the **demo** endpoint URL to your application
```

Comment:
`Add the **demo** endpoint URL as a webhook to your application of choice (Atlas or Github)`